firmware/controllers/can/FOME_CAN_verbose.dbc

```diff
@@ -1,7 +1,7 @@
-VERSION ""
+VERSION "HIPBNYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY/4/%%%/4/'%**4YYY///"
```

Comment:
what happened here...?

what version of kvaser database did you use?

Reply:
2.5.475, no Idea, let me revert that

Reply:
update!
![image](https://github.com/FOME-Tech/fome-fw/assets/568254/e86153fd-5091-45ad-847c-a7ad7f556f4a)